definitions/features/upstream_repositories.rb

```diff
@@ -0,0 +1,23 @@
+class Features::UpstreamRepositories < ForemanMaintain::Feature
+  metadata do
+    label :upstream_repositories
+    description 'Feature for operations on upstream repositories'
+  end
+
+  def repoids_and_urls
```

Comment:
I would have called this `enabled_upstream_repos` to match the `enabled_repos` of the `repository_manager`, but this might be just me ;)